pkg/api/middlewares/authentication.go

```diff
@@ -0,0 +1,95 @@
+package middlewares
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"net/http"
+	"slices"
+	"strings"
+	"time"
+
+	"github.com/nais/knorten/pkg/database"
```

Comment:
```suggestion
	"github.com/navikt/knorten/pkg/database"
```
Her er det en liten opprydding vi/jeg har glemt. Knorten har byttet org fra `nais` til `navikt`.